server/ai_live_video.go

```diff
@@ -141,6 +142,17 @@ func startTricklePublish(ctx context.Context, url *url.URL, params aiRequestPara
 	clog.Infof(ctx, "trickle pub")
 }
 
+func suspendOrchestrator(ctx context.Context, params aiRequestParams) {
+	sel, err := params.sessManager.getSelector(ctx, core.Capability_LiveVideoToVideo, params.liveParams.pipeline)
+	if err != nil {
+		clog.Warningf(ctx, "Error suspending orchestrator: %v", err)
+		return
+	}
+	// We do selection every 6 min, so it effectively means the Orchestrator won't be selected for the next 30 min (unless there is no other O available)
+	penalty := 5
+	sel.suspender.suspend(params.liveParams.sess.Transcoder(), penalty)
```

Comment:
It would be nice to print out a log saying which O got suspended and what's the penalty.

Reply:
Added in 84bf8044b29a0b0a6173f8e976b056a200f2397b